### ISSUE TITLE
Attendance Coins

### ISSUE THREAD

**Issue by Alchemist21** [mickaelgermemont/AttendanceCoin#3]
For late attendees, will there be something like a coin with a lower value? How will the valuation be like?

**Comment by Adrianf23** [comment]
I think it should be based on contribution given during the class. That way, even if you are late and engage, you are still able to reap rewards. That is essentially the same thing as what you said, but consistent attendees will be able to complete more in a shorter timeframe than someone who occasionally comes.

**Comment by Adrianf23** [comment]
I was also going to add that effort is rewarded. So attendance is not necessarily a physical thing

**Comment by vutsalsinghal** [comment]
@Alchemist21 I think it should not matter if you're late. As long as you are contributing meaningfully even after coming late class is fine... So I agree with @Adrianf23

**Comment by vutsalsinghal** [comment]
@Alchemist21 I think it should not matter if you're late. As long as you are contributing meaningfully even after coming late class is fine... So I agree with @Adrianf23

**Comment by tesla809** [comment]
Perhaps we can abstract attendance to "contribution". Attendance is one way to contribute. However, you can contribute in other ways, such as contributing to open source projects, answering questions on slack, creating medium posts, etc. What do you guys think?

**Comment by mickaelgermemont** [comment]
i agree. i also like the idea of getting AttendanceCoins as a contribution reward.
